core/arch/arm/kernel/stmm_sp.c

```diff
@@ -631,24 +723,69 @@ static TEE_Result sec_storage_obj_write(unsigned long storage_id, char *obj_id,
 
 static bool stmm_handle_mem_mgr_service(struct thread_svc_regs *regs)
```

Comment:
This looks a bit clumsy. I realize that it's a bit tricky to make common code here, but it seems like a lot of effort to have a common call to `service_compose_direct_resp(regs, stmm_rc)` followed by a `return true`.

Reply:
yes, i've [something](https://github.com/etienne-lms/optee_os/commit/b47a3b742bd5558bf1e84ec5ecbeb3099d8a05e9) for that. Maybe I can add it to this series.